lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.spc_typ.json

```diff
@@ -5201,10 +5201,10 @@
       "occupancy_schedule": "Hospital BLDG_OCC_EXTD_SCH",
       "occupancy_activity_schedule": "Hospital ACTIVITY_SCH",
       "is_residential": null,
-      "ventilation_standard": "ASHRAE 62.1-2007",
+      "ventilation_standard": "ASHRAE 62.1-2016",
       "ventilation_primary_space_type": "Food and Beverage Service",
-      "ventilation_secondary_space_type": "Cafeteria/fast food dining",
-      "ventilation_per_area": 0.18,
+      "ventilation_secondary_space_type": "Kitchen (cooking)",
+      "ventilation_per_area": 0.12,
```

Comment:
The ventilation standard and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.